src/CollapseTable.css

```diff
@@ -0,0 +1,140 @@
+      </div>
+      <table class='infobox vcard'
+        style='width: 22em; margin-top: 0px; margin-bottom: 0px; display: none'>
+        <tbody>
+          <tr>
+            <th colspan=2
+              style='text-align: center; font-size: 125%; font-weight: bold; font-size: 130%'>
+              <span class=fn>Barack Obama</span>
+            </th>
+          </tr>
+          ...
+        </tbody>
+      </table>
+      <div class='app_table_collapsed_bottom app_table_collapse_icon'
+        style='display: none'>
+        Close
+      </div>
+    </div>
+
+  - Frankenstein's monster's expanded infobox:
+
+    <div class=app_table_container>
+      <div class='app_table_collapsed_container app_table_collapse_close app_table_collapse_icon'
+        style='display: block'>
+        <strong>Quick facts</strong><span class=app_span_collapse_text>: Created
+          by, Information …</span>
+      </div>
+      <table class=infobox
+        style='width: 22em; border-spacing: 2px 5px; margin-top: 0px; margin-bottom: 0px; display: block'>
+        <tbody>
+          <tr>
+            <th colspan=2
+              style='text-align: center; font-size: 125%; font-weight: bold; background: #dedee2;'>
+              Frankenstein's monster
+            </th>
+          </tr>
+          ...
+        </tbody>
+      </table>
+      <div class='app_table_collapsed_bottom app_table_collapse_icon'
+        style='display: block'>
+        Close
+      </div>
+    </div>
+
+  - Dracula's collapsed notes and references:
+
+    <div class=app_table_container>
+      <div class='app_table_collapsed_container app_table_collapsed_open'
+        style='display: block'>
+        <strong class=app_table_collapsed_caption>Tap to expand</strong>
+      </div>
+      <div class='reflist references-column-width'
+        style='column-width: 30em; list-style-type: decimal; padding: 4px; display: none'>
+          ...
+      </div>
+      <div class='app_table_collapsed_bottom app_table_collapse_icon'
+        style='display: none'>
+        Close
+      </div>
+    </div>
+*/
+
+.content table.infobox {
+  font-size: 100%;
+  border-style: none !important;
+}
+
+table.infobox caption {
+  text-align: center;
+  font-weight: bold;
+}
+
+/* The outermost table container. todo: rename. */
+.app_table_container {
+  width: 100%;
+  margin-top: 14px;
+  margin-bottom: 14px;
+  border-radius: 2px;
+  box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.12), 0 0 1px 0 rgba(0, 0, 0, 0.18);
+  clear: both;
+}
+
+/* The outermost table container's header. todo: rename. */
+.app_table_collapsed_container {
+  width: 100%;
+  padding: 12px 48px 12px 12px;
+  line-height: 120%;
+  background-color: #f8f8f8;
+  background-repeat: no-repeat;
+  background-position: 95% 50%;
+  background-size: 16px 16px;
+}
+
+/* The outermost table container header's subcaption text. todo: rename. */
+.app_span_collapse_text {
+  color: #808080;
```

Comment:
These colors differ from iOS. I'm not sure any of these styles should be here in the long term (maybe just empty classes or classes with visibility changes).